en/installation.rst

```diff
@@ -16,6 +16,21 @@ Requirements
 - mbstring extension
 - intl extension
 
+.. note::
+
+    In both XAMPP and WAMP, mcrypt and mbstring extensions are working by default.
+
+    In XAMPP, intl extension is included but you have to uncomment
+    extension=php_intl.dll in php.ini and restart the server through the XAMPP
+    Control Panel.
+
+    In WAMP, the intl extension is "activated" by default but not working.
+    To make it work you have to go to php folder (by default)
+    C:\wamp\bin\php\php5.5.12, copy all the files that looks like icu***.dll
```

Comment:
Also, the paths should maybe be in the `double backticks`, same as filenames etc.